py/mockbuild/util.py

```diff
@@ -631,7 +631,7 @@ def _prepare_nspawn_command(chrootPath, user, cmd, nspawn_args=[], env=None, cwd
             raise exception.Error('Internal Error: command must be list or shell=True.')
     elif not cmd_is_list:
         cmd = [cmd]
-    nspawn_argv = ['/usr/bin/systemd-nspawn', '-q', '-M', uuid.uuid4().hex, '-D', chrootPath]
+    nspawn_argv = ['/usr/bin/systemd-nspawn', '-q', '-a', '-M', uuid.uuid4().hex, '-D', chrootPath]
```

Comment:
would it be worth adding a comment about why we are running as pid2? could be useful for future contributors to the code.